src/lib/structures/expr.ml

```diff
@@ -2584,7 +2610,7 @@ module Purification = struct
         mk_term (Sy.Var fresh_var) [] t.ty , add_let fresh_var t lets
 
       | _ -> (* detect ITEs *)
-        match t.xs with
+        match tm.xs with
```

Comment:
Is this change intentional?

Reply:
Given that the CI fails on this I'll guess it was not :rofl:

Reply:
Nope, there is no code change in a documentation PR ;)